evap/results/tools.py

```diff
@@ -296,7 +296,8 @@ def textanswers_visible_to(contribution):
         ).distinct().order_by('last_name', 'first_name')
     else:
         contributors = [contribution.contributor]
-    num_delegates = len(set(UserProfile.objects.filter(represented_users__in=contributors).distinct()) - set(contributors))
+    non_proxy_contributors = [contributor for contributor in contributors if not contributor.is_proxy_user]
+    num_delegates = len(set(UserProfile.objects.filter(represented_users__in=non_proxy_contributors).distinct()) - set(contributors))
```

Comment:
I think this changes the character of the method so much that it should be renamed (or copied, in case the old method is still needed somewhere) -- and subsequently also the names of the variables where the results are stored should be renamed...

Reply:
Technically all users are still included in the result, because the proxy users are 'composed' of their delegates. So even though they are deducted from the delegate count one still knows they can see the text answer, because the proxy user they represent is listed as contributor. So  no copying or renaming should be necessary